libjulia/backends/evm/EVMCodeTransform.cpp

```diff
@@ -65,7 +65,9 @@ void CodeTransform::operator()(const FunctionDefinition&)
 	solAssert(false, "Function definition not removed during desugaring phase.");
 }
 
-void CodeTransform::generateAssignment(Identifier const& _variableName, SourceLocation const& _location)
+void CodeTransform::generateAssignment(
+	Identifier const& _variableName,
+	SourceLocation const& _location)
```

Comment:
Not at all.  The funny thing is that I don't remember doing that in the first place.  The line width isn't so long as to really need to span multiple lines.  I'll just put it back the way it was.